common.mk

```diff
@@ -1,12 +1,35 @@
 # This makefile is intended to be included by each package's makefile.  The
 # paths are relative to the package directory.
 
-ROOT := $(CURDIR)/..
+ROOT := $(dir $(lastword $(MAKEFILE_LIST)))
 SOURCES := $(wildcard src/*)
 VERSION := $(shell node -pe "require('./package.json').version")
+DOC_DESTINATION := $(subst @fluent, ../html, $(PACKAGE))
 
 export SHELL := /bin/bash
-export PATH  := $(CURDIR)/node_modules/.bin:$(ROOT)/node_modules/.bin:$(PATH)
+ESLINT ?= $(ROOT)node_modules/.bin/eslint
+TSC ?= $(ROOT)node_modules/.bin/tsc
+NYC ?= $(ROOT)node_modules/.bin/nyc
+MOCHA ?= $(ROOT)node_modules/.bin/mocha
+ROLLUP ?= $(ROOT)node_modules/.bin/rollup
+TYPEDOC ?= $(ROOT)node_modules/.bin/typedoc
+
+ROLLUP_CMD = $(ROLLUP) $(CURDIR)/esm/index.js \
+	--banner "/* $(PACKAGE)@$(VERSION) */" \
+	--amd.id $(PACKAGE) \
+	--name $(GLOBAL) \
+	--output.format umd \
+	--output.file \
+	$(NULL)
```

Comment:
What is the `$(NULL)` for?

Reply:
"Best practices", they said. https://searchfox.org/mozilla-central/search?q=%24%28NULL%29&path=&case=false&regexp=false is where I got this from.

Also allows you to add a line at the end w/out having to change the line before, as they all end in `\`.

Reply:
Thanks, that makes sense.